tools/pv_tree_generator/build_tree_test.py

```diff
@@ -27,7 +27,8 @@ class BuildTreeTest(unittest.TestCase):
     @staticmethod
     def get_sv():
         """pick a subset of stat_var dcids"""
-        level0 = ['MarriedPopulation', 'DivorcedPopulation', 'MalePopulation']
+        level0 = ['MarriedPopulation', 'DivorcedPopulation',
```

Comment:
for future PR, can update test to use the new stats var as well

Reply:
will do